fuel-vm/src/tests/blockchain.rs

```diff
@@ -1319,13 +1319,21 @@ fn smo_instruction_works() {
         let secret = SecretKey::random(rng);
         let sender = rng.gen();
 
+        // Two bytes of random data to send as the message
+        let msg_data = [rng.gen::<u8>(), rng.gen::<u8>()];
+
         #[rustfmt::skip]
         let script = vec![
+            op::movi(0x10, 2),                          // data buffer allocation size
```

Comment:
Maybe with `script_with_data_offset!` we may do it more readable=D

Reply:
This is not script_data, instead I just allocate heap space for the bytes. Do you think I should rather use script_data for this?

Reply:
We probably shouldn't use script_with_data_offset anymore now that we have `op::gtf` to find the start of the script data.